Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -12,157 +11,97 @@ To get started with the Mixed Reality Toolkit you will need:
 
-### 2. [Import the Mixed Reality Toolkit asset](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases)
+7. You will get a prompt asking you to import "TMP Essentials". 
 
-> The Mixed Reality Toolkit is available via [multiple delivery mechanisms](DownloadingTheMRTK.md) and in the future will also be available via the Unity package manager once Unity makes that option available.
+![TMP Essentials](../External/Documentation/Images/MRTK_GettingStarted_TMPro.png)
 
-Get the asset and import it in to your Unity project using  "Asset -> Import Package -> Custom Package" from the Unity Editor menu.
+8. Select "Import TMP essentials" button. "TMP Essentials" refers to TextMeshPro plugin, which some of the MRTK examples use for improved text rendering.
 
-![](../External/ReadMeImages/Unity_ImportAssetOption.png)
+9. Close the TMPPro dialog. After this you need to reload Unity, so close unity and re-open unity, then open your project and go back to the scene.
 
-Accept all the content and continue.
+10. Finally, press the play button to start the scene and explore the scene! You can use simulated hands to interact in editor. Press WASD keys to fly around the scene. Press and hold right mouse to look around. Press and hold space bar to use a simulated hand.
 
-![](../External/ReadMeImages/MRTK_AssetImportDialog.png)
+There's quite a bit to explore in this scene, have fun! Learn more about the components in the scene [in the hand interaction examples guide](README_HandInteractionExamples.md). You can also read through [input simulation docs](InputSimulation/InputSimulationService.md) to learn more about in-editor hand input simulation in MRTK.
 
-> The SDK is optional but highly recommended for new users.  Once you have a feel for how the toolkit works, you can remove these safely if you are not using them.
+Congratulations, you just used your first MRTK scene. Now onto creating your own stuff...
 
-> The MRTK also provides a [separate "Examples" asset](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases), which currently contains demonstration scenes for current MRTK features.  In the future this will also contain educational "Example" scenes to aid with learning implementation tricks with the toolkit.
+## Add MRTK to a new scene or new project
 
-Note that some prefabs and assets require TextMesh Pro, meaning you have to have the TextMesh Pro package installed and the assets in your project (Window -> TextMeshPro -> Import TMP Essential Resources).
+1. Create a new project, or start a new scene in your current project
 
-### 3. Accept the Setup prompt (optional)
-![](../External/ReadMeImages/MRTK_UnitySetupPrompt.png)
+> *Note* when creating a new project with Unity 2018, Unity gives you several templates to choose from.  Currently the **MRTK does not yet support the Scriptable Render Pipeline**, so the LWSRP, HDSRP and VRSRP projects are not compatible with MRTK projects.
```

Comment:
>LWSRP, HDSRP and VRSRP  [](start = 188, length = 23)

Is it expected for the reader to know what these stand for? I personally don't.

Reply:
No, I didn't know what they meant either. Updated to jus say "create using 3D template"

---
In reply to: [273692256](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3904#discussion_r273692256) [](ancestors = 273692256)